slidechain/postpegout.go

```diff
@@ -15,32 +15,35 @@ import (
 	"github.com/stellar/go/xdr"
 )
 
-func (c *Custodian) doPostPegOut(ctx context.Context, assetXDR, anchor, txid []byte, amount, seqnum int64, peggedOut pegOutState, exporter, tempAddr string, pubkey []byte) error {
+func (c *Custodian) doPostPegOut(ctx context.Context, p pegOut, txid []byte) error {
```

Comment:
See my other comment about keeping txid out of the json serialization of `pegOut`. If you do that, you won't need a separate `txid` argument here.